src/plugins/hetero/async_infer_request.cpp

```diff
@@ -57,6 +57,13 @@ StatusCode HeteroAsyncInferRequest::Wait(int64_t millis_timeout) {
     return waitStatus;
 }
 
+InferenceEngine::Blob::Ptr HeteroAsyncInferRequest::GetBlob(const std::string& name) {
+    CheckState();
+    auto blob = _heteroInferRequest->GetBlob(name);
+    setPointerToSo(_heteroInferRequest->getPointerToSo());
+    return blob;
```

Comment:
do we need to override async request? I suppose sync one already returns "fixed" blob, is not it?

Reply:
Thanks Ilya. create_infer_request() is returning async request, getblob is calling sync request to getblob, here I want to get the additional so dependency(if the individual plugin sets one) from the async request.